system_reports/MACOS/M1/osx-xcode-14.0.x-ventura.log

```diff
@@ -220,49 +220,49 @@ nettle 3.8.1
 npth 1.6
 oniguruma 6.9.8
 openconnect 9.01_2
-openexr 3.1.6
-openjdk 19.0.2
+openexr 3.1.7
+openjdk 20
```

Comment:
Is it intended?

Reply:
Yup, we use the `openjdk` cask and not `openjdk@19` for installing the latest. So this is expected and I think it's good practice for installing the always latest.